src/vueWrapper.ts

```diff
@@ -30,6 +30,9 @@ export class VueWrapper<T extends ComponentPublicInstance> {
     this.rootVM = vm?.$root
     this.componentVM = vm as T
     this.__setProps = setProps
+
+    this.attachNativeEventListener()
+
```

Comment:
Perhaps this method should be exported from `emit.ts` with `this` as an argument, the style looks very different from anything else so I tried to add a private method instead. There is definitely some bleeding of concerns.